python/test/unit/mesh/test_mesh_utils.py

```diff
@@ -1,393 +0,0 @@
-# Copyright (C) 2020 Garth N. Wells
```

Comment:
Is this removal related to PR?

Reply:
Yes - it used some removed functionality. It was intended for removal at some point.